### ISSUE TITLE
update dependancies for the kit

### ISSUE THREAD

**Issue by BillyG83** [web3ui/web3uikit#676]
there is a LOT of warnings, we should check and update all our dependancies

```
warning " > @storybook/theming@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning " > @storybook/theming@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning " > @web3auth/web3auth@0.3.3" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @toruslabs/openlogin@1.7.3" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/base@0.3.1" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @toruslabs/openlogin > @toruslabs/http-helpers@2.2.0" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @toruslabs/openlogin > @toruslabs/openlogin-jrpc@1.7.3" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @toruslabs/openlogin > @toruslabs/openlogin-utils@1.7.0" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/openlogin-adapter > @toruslabs/openlogin-ed25519@1.7.0" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/openlogin-adapter > @web3auth/base-provider@0.3.1" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/openlogin-adapter > @web3auth/ethereum-provider@0.3.1" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/openlogin-adapter > @web3auth/solana-provider@0.3.1" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/torus-evm-adapter > @toruslabs/torus-embed@1.22.0" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/torus-solana-adapter > @toruslabs/solana-embed@0.1.6" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/openlogin-adapter > @web3auth/base-provider > @toruslabs/base-controllers@1.6.14" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/torus-evm-adapter > @toruslabs/torus-embed > @toruslabs/fetch-node-details@5.0.1" has unmet peer dependency "@babel/runtime@7.x".
warning "@web3auth/web3auth > @web3auth/torus-evm-adapter > @toruslabs/torus-embed > @toruslabs/torus.js@5.1.0" has unmet peer dependency "@babel/runtime@7.x".
warning "react-moralis > use-immer@0.6.0" has incorrect peer dependency "react@^16.8.0 || ^17.0.1".
warning " > wp5-cra-polyfill@0.0.1" has unmet peer dependency "webpack@>=5".
warning "wp5-cra-polyfill > node-polyfill-webpack-plugin@1.1.4" has unmet peer dependency "webpack@>=5".
warning " > @storybook/addons@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning " > @storybook/addons@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/api@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/api@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/components@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/components@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > react-inspector@5.1.1" has incorrect peer dependency "react@^16.8.4 || ^17.0.0".
warning "@storybook/addon-links > @storybook/router@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-links > @storybook/router@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/components > react-popper-tooltip@3.1.1" has incorrect peer dependency "react@^16.6.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/components > react-popper-tooltip@3.1.1" has incorrect peer dependency "react-dom@^16.6.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/components > react-textarea-autosize@8.3.3" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/components > react-popper-tooltip > react-popper@2.2.5" has incorrect peer dependency "react@^16.8.0 || ^17".
warning "@storybook/addon-actions > @storybook/components > react-textarea-autosize > use-composed-ref@1.2.1" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-actions > @storybook/components > react-textarea-autosize > use-latest@1.2.0" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-controls > @storybook/core-common@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-controls > @storybook/core-common@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-controls > @storybook/store@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-controls > @storybook/store@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @mdx-js/react@1.6.22" has incorrect peer dependency "react@^16.13.1 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/builder-webpack4@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/builder-webpack4@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core@6.4.21" has unmet peer dependency "webpack@*".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/preview-web@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/preview-web@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/source-loader@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/source-loader@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-info > react-element-to-jsx-string@14.3.4" has incorrect peer dependency "react-dom@^0.14.8 || ^15.0.1 || ^16.0.0 || ^17.0.1".
warning " > babel-loader@8.2.4" has unmet peer dependency "webpack@>=2".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/builder-webpack4 > @storybook/client-api@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/builder-webpack4 > @storybook/client-api@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/builder-webpack4 > @storybook/ui@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/builder-webpack4 > @storybook/ui@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core > @storybook/core-client@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core > @storybook/core-client@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core > @storybook/core-client@6.4.21" has unmet peer dependency "webpack@*".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core > @storybook/core-server@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core > @storybook/core-server@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core > @storybook/core-server > @storybook/manager-webpack4@6.4.21" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning "@storybook/addon-essentials > @storybook/addon-docs > @storybook/core > @storybook/core-server > @storybook/manager-webpack4@6.4.21" has incorrect peer dependency "react-dom@^16.8.0 || ^17.0.0".
warning "@storybook/addon-info > @storybook/addons > @storybook/api@5.3.21" has unmet peer dependency "regenerator-runtime@*".
warning "@storybook/addon-info > @storybook/addons > @storybook/api > @reach/router@1.3.4" has incorrect peer dependency "react-dom@15.x || 16.x || 16.4.0-alpha.0911da3".
warning " > @storybook/react@6.5.9" has unmet peer dependency "require-from-string@^2.0.2".
warning " > @storybook/testing-react@1.2.4" has unmet peer dependency "@storybook/client-api@>=6.4.0".
warning " > @storybook/testing-react@1.2.4" has unmet peer dependency "@storybook/preview-web@>=6.4.0".
warning " > @storybook/testing-react@1.2.4" has incorrect peer dependency "react@^16.8.0 || ^17.0.0".
warning " > jest-svg-transformer@1.0.0" has unmet peer dependency "jest@>22".
warning " > react-test-renderer@17.0.2" has incorrect peer dependency "react@17.0.2".
warning " > rollup-plugin-inline-svg@2.0.0" has unmet peer dependency "rollup@*".
```

https://media.giphy.com/media/QMHoU66sBXqqLqYvGO/giphy.gif

**Comment by BillyG83** [comment]
hey @AbhinavMV are you working on this? its not priority but if you have picked it up please remove the `ready for dev` and add the `in progress` label. TY dude 😁

**Comment by AbhinavMV** [comment]
closed by #702
